README.md

```diff
@@ -1,75 +1,101 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# HTML & CSS3 & JavaScript Course
 
-# Project Name
+If you are not familiar with linters, read [root level README](../README.md).
 
-> Description the project.
+## Set-up GitHub Actions
 
```

Comment:
Kindly replace your Readme file with [Microverse Readme Template](https://github.com/microverseinc/readme-template) and customize it according to your Project 🤗

Also, please make sure that your README contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).